CSS lists and counters: update landing page (#30174)
mdn/content: files/en-us/web/css/css_lists/index.md

{{CSSRef}}

> The **CSS lists and counters** module enables styling the bullets of list items and their counters, including styling, positioning, and manipulating their values, including incrementing counters by values other than `1`.

Reviewer: The sentence reads a bit strangely. You style bullets, sure, but you don't style counters. They are used to modify the values shown in the bullets? You've also got "...including ... including...".

Not 100% sure if this is correct, but can I suggest something like

```suggestion
The **CSS lists and counters** module enables styling and positioning of list item bullets and manipulating their values with a combination of strings, counters, and other features.
```

> A list item's marker, whether a bullet symbol or ordinal counter, is its defining feature. List items are not limited to {{HTMLElement("li")}} elements nested within an {{HTMLElement("ol")}} or {{HTMLElement("ul")}}. Rather, lists items are element with `display: list-item` set.

Reviewer: ```suggestion
A list item's marker, whether a bullet symbol or ordinal counter, is its defining feature. List items are not limited to {{HTMLElement("li")}} elements nested within {{HTMLElement("ol")}} or {{HTMLElement("ul")}} elements. Rather, list items are any element with `display: list-item` set.
```

> This module defines the CSS to set and reset a list's counters, to set which [counter-style](/en-US/docs/Web/CSS/CSS_counter_styles) or symbol is used as the marker, and position those markers, and provides developers with the ability to create customized markers.

Reviewer: ```suggestion
This module defines CSS features to set and reset a list's counters, set which [counter-styles](/en-US/docs/Web/CSS/CSS_counter_styles) or symbols to use as its markers, and position those markers. It also provides developers with the ability to create customized markers.
```

## Reference

### Properties

- {{cssxref("counter-increment")}}
- {{cssxref("counter-reset")}}
- {{cssxref("counter-set")}}
- {{cssxref("list-style-image")}}
- {{cssxref("list-style-type")}}
- {{cssxref("list-style-position")}}
- {{cssxref("list-style")}} shorthand

> There is also a `marker-side` property, which is yet to be fully defined or implemented.

Author: ```suggestion
> **Note:** The CSS lists and counter module introduces one at-risk property that has not been fully defined or implemented: `marker-side`.
```
if you prefer this way of writing it, feel free to commit this edit. This matches the note on the CSS generated content module page.

> There is also a `marker-side` property, which is yet to be fully defined or implemented.

Reviewer: Maybe you could put this in a note, and add a brief description of what this property's functionality is meant to be? As it stands, it isn't useful to include this. 

You could remove it anyway; the page could become outdated if it is removed from the spec or changed significantly.

Author: This page is an overview of the spec. It's been in the spec for 10 years. There's almost nothing about it online. But, if someone does see it in the spec, and does a search, it's good for them to be able to learn "it's not implemented". We need to include it, and this seemed like the best spot and way to do so.

The reason I put this note in (but not in the form of a note) is:
1) if people see it in the spec then look for it on MDN, at least it's mentioned and findable
2) If ever added, the person adding it knows to link it from here (and add it to the list) - this is basically a placeholder saying "add me here when ready.
3) It's been in the spec since before 2014 and is unlikely to get removed. (talked to a spec author about it).
4) not a note because notes are for important things, and this is not important.

The reason it's not defined here is:
1) no properties are defined on this page
2) the property name is fully descriptive, literally what side the marker is on

Reviewer: OK, this seems like a reasonable justification to me.
